node/src/actors/chain_manager/mod.rs

```diff
@@ -744,7 +749,7 @@ impl ChainManager {
                     actix::fut::ok(())
                 }
                 Err(e) => {
-                    log::warn!("{}", e);
+                    log::warn!("Validate transacion {}: {}", msg.transaction.hash(), e);
```

Comment:
I wouldn't be able to wrap my head around a warning saying "validate transaction". Is it telling me to go and validate it by hand? :stuck_out_tongue: 

Maybe something like this?
```suggestion
                    log::warn!("Error when validating transaction {}: {}", msg.transaction.hash(), e);
```